charts/ingressmonitorcontroller/templates/deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
         - containerPort: 8443
           name: https
       {{- end }}
+        resources:
+          {{- toYaml .Values.resources | nindent 10 }}
```

Comment:
what if the rbac proxy resources are different from the manage container? plz refer to the Forecastle chart